tests/integration/tables/interface_details.cpp

```diff
@@ -26,6 +28,11 @@ class InterfaceDetailsTest : public testing::Test {
 };
 
 TEST_F(InterfaceDetailsTest, test_sanity) {
+#ifdef OSQUERY_WINDOWS
+  LOG(INFO) << "Test failing on Windows, temporarily disabled";
```

Comment:
I'd wish we had a consistent pattern for this, something to more easily look for. But this seems fine

Reply:
We can eventually have a function in some existing test helper code that checks for a given list of platforms and prints a standard message.

Obviously there are a couple of other ways to disable tests, which though either disable entirely a single test case or a test suite.
For googletest this is prepending DISABLED_ to the test case name (for instance `TEST_F(InterfaceDetailsTest, DISABLED_test_sanity`) or test suite name (for instance `class DISABLED_InterfaceDetailsTest : public testing::Test {`)